### ISSUE TITLE
Scaling laws for neural language models

### ISSUE THREAD

**Issue by irthomasthomas** [irthomasthomas/undecidability#507]
- [ ] [Scaling laws for neural language models](https://openai.com/research/scaling-laws-for-neural-language-models)

# TITLE: Scaling laws for neural language models

## DESCRIPTION:
Abstract
We study empirical scaling laws for language model performance on the cross-entropy loss. The loss scales as a power-law with model size, dataset size, and the amount of compute used for training, with some trends spanning more than seven orders of magnitude. Other architectural details such as network width or depth have minimal effects within a wide range. Simple equations govern the dependence of overfitting on model/dataset size and the dependence of training speed on model size. These relationships allow us to determine the optimal allocation of a fixed compute budget. Larger models are significantly more sample-efficient, such that optimally compute-efficient training involves training very large models on a relatively modest amount of data and stopping significantly before convergence.

URL: https://openai.com/research/scaling-laws-for-neural-language-models

#### {   "label-name": "Scaling-Laws",   "description": "Analysis of scaling laws for language models",   "confidence": 69.89 }
